### PR TITLE
Simplified checking result in flow tests, fixed traffic generated test

### PR DESCRIPTION
**Pull request by ValentinTyhonov**
Simplified checking result in flow tests in case when few expected aggregated objects are in one JSON file 

Fixed traffic generated test - did not work before

### CODE REVIEW

File: src/test/java/com/ericsson/ei/flowtests/FlowTest2.java [hunk 1]
```diff
@@ -68,10 +70,10 @@ String setEventsFilePath() {
...
 
     @Override
-    Map<String, String> setCheckInfo() {
-        Map<String, String> checkInfo = new HashMap<>();
-        checkInfo.put(AGGREGATED_OBJECT_ID_1, AGGREGATED_OBJECT_FILE_PATH_1);
-        checkInfo.put(AGGREGATED_OBJECT_ID_2, AGGREGATED_OBJECT_FILE_PATH_2);
-        return checkInfo;
+    Map<String, JsonNode> setCheckData() throws IOException {
+        Map<String, JsonNode> checkData = new HashMap<>();
+        checkData.put(AGGREGATED_OBJECT_ID_1, getJSONFromFile(AGGREGATED_OBJECT_FILE_PATH_1));
```
Comment (evasiba-ericsson) [inline src/test/java/com/ericsson/ei/flowtests/FlowTest2.java]: put value from getJsonFromFile in a variable like  you did in the FlowTest

File: src/test/java/com/ericsson/ei/flowtests/FlowTest.java [hunk 2]
```diff
@@ -97,9 +98,10 @@ String setEventsFilePath() {
     }
 
     @Override
-    Map<String, String> setCheckInfo() {
-        Map<String, String> checkInfo = new HashMap<>();
-        checkInfo.put(AGGREGATED_OBJECT_ID, AGGREGATED_OBJECT_FILE_PATH);
-        return checkInfo;
+    Map<String, JsonNode> setCheckData() throws IOException {
```
Comment (evasiba-ericsson) [inline src/test/java/com/ericsson/ei/flowtests/FlowTest.java]: you do not set anything here but get the data so the name should be getExpectedData

File: src/test/java/com/ericsson/ei/flowtests/FlowTestBase.java [hunk 3]
```diff
@@ -136,15 +135,13 @@ private void waitForEventsToBeProcessed(int eventsCount) {
...
+        checkData.forEach((id, expectedJSON) -> {
             try {
                 String document = objectHandler.findObjectById(id);
-                String expectedDocument = FileUtils.readFileToString(new File(filePath), "UTF-8");
-                JsonNode expectedJson = objectMapper.readTree(expectedDocument);
-                JsonNode actualJson = objectMapper.readTree(document);
-                LOGGER.info("Complete aggregated object: " + actualJson);
-                JSONAssert.assertEquals(expectedJson.toString(), actualJson.toString(), false);
+                JsonNode actualJSON = objectMapper.readTree(document);
+                LOGGER.info("Complete aggregated object: " + actualJSON);
```
Comment (evasiba-ericsson) [inline src/test/java/com/ericsson/ei/flowtests/FlowTestBase.java]: switch to logger.debug

File: src/test/java/com/ericsson/ei/flowtests/TrafficGeneratedTest.java [hunk 4]
```diff
@@ -189,40 +118,31 @@ public void trafficGeneratedTest() {
...
-            for (int i = 0; i < EVENT_PACKAGES; i++) {
-                String document = objectHandler.findObjectById("6acc3c87-75e0-4b6d-88f5-b1a5d4".concat(String.format("%06d", i)));
-                JsonNode actualJson = objectmapper.readTree(document);
-                assertEquals(expectedJson.toString().length(), actualJson.toString().length());
-            }
+            checkResult();
 
             String time = "" + diffTime / 60000 + "m " + (diffTime / 1000) % 60 + "s " + diffTime % 1000;
-            System.out.println("Time of execution: " + time);
+            LOGGER.info("Number of events, that were sent: " + eventsCount);
```
Comment (evasiba-ericsson) [inline src/test/java/com/ericsson/ei/flowtests/TrafficGeneratedTest.java]: change to logger.debug

File: src/test/java/com/ericsson/ei/flowtests/FlowTestExternalComposition.java [hunk 5]
```diff
@@ -58,8 +60,12 @@ String setEventsFilePath() {
     }
 
     @Override
-    Map<String, String> setCheckInfo() {
-        Map<String, String> checkInfo = new HashMap<>();
-        return checkInfo;
+    Map<String, JsonNode> getCheckData() throws IOException {
```
[the remark this answers is not in the document]
Comment (ValentinTyhonov) [reply src/test/java/com/ericsson/ei/flowtests/FlowTestExternalComposition.java]: As I understand this test was not workable. I left it for future. In other case we can remove this test.

[the remark this answers is not in the document]
Comment (evasiba-ericsson) [reply src/test/java/com/ericsson/ei/flowtests/FlowTestExternalComposition.java]: ok. remove it

**Pull request merged**